### PR TITLE
Includes, delegates

### PR DESCRIPTION
**Pull request by tenhobi**
- [x] functionality - modules, includes, delegates, ...
- [x] update documentation

### CODE REVIEW

File: packages/auto_mappr/lib/src/builder/auto_mappr_builder.dart [hunk 1]
```diff
@@ -140,6 +131,7 @@ class AutoMapprBuilder {
 
       // Generates nullable mapping method only when nullable method is used.
       // TODO(later): switch to MappingMethodBuilder.
```
Comment (petrnymsa) [inline packages/auto_mappr/lib/src/builder/auto_mappr_builder.dart:133]: Is this todo still relavant?

File: packages/auto_mappr/lib/src/builder/map_bodies/class_body_builder.dart [hunk 2]
```diff
@@ -158,9 +157,9 @@ class ClassBodyBuilder extends MapBodyBuilderBase {
         final targetField =
             (mapping.target).getAllGetters().firstWhereOrNull((field) => field.displayName == param.displayName);
 
-        final fieldMapping = mapping.tryGetFieldMapping(param.displayName);
+        final fieldMappingX = mapping.tryGetFieldMapping(param.displayName);
```
Comment (petrnymsa) [inline packages/auto_mappr/lib/src/builder/map_bodies/class_body_builder.dart:160]: Can we name this variable differently? In outer scope we have already fieldMapping.

File: packages/auto_mappr/lib/src/models/auto_mappr_config.dart [hunk 3]
```diff
@@ -46,31 +46,44 @@ class AutoMapprConfig {
...
 
-  List<String> _modulesDocComment() {
+  List<String> _importsDocComment() {
     return [
-      if (modulesList.isNotEmpty) ...[
+      if (importsList.isNotEmpty) ...[
         '///',
-        "/// Available modules: ${modulesList.map((e) {
-          final alias = e.type!.element!.getLibraryAlias(config: this);
+        "/// Used imports: ${importsList.map((e) {
```
Comment (petrnymsa) [inline packages/auto_mappr/lib/src/models/auto_mappr_config.dart:58]: Wording: "Used includes"

File: packages/auto_mappr/lib/src/models/auto_mappr_config.dart [hunk 4]
```diff
@@ -46,31 +46,44 @@ class AutoMapprConfig {
...
+      ..._importsDocComment(),
+      ..._delegatesDocComment(),
     ];
   }
 
-  List<String> _modulesDocComment() {
+  List<String> _importsDocComment() {
     return [
-      if (modulesList.isNotEmpty) ...[
+      if (importsList.isNotEmpty) ...[
```
Comment (petrnymsa) [inline packages/auto_mappr/lib/src/models/auto_mappr_config.dart:56]: importsList should be named "includesList" to be concise

**Review by petrnymsa — CHANGES_REQUESTED** [review]

File: packages/auto_mappr/lib/src/models/auto_mappr_config.dart (hunk 3 above)
Comment (tenhobi) [reply packages/auto_mappr/lib/src/models/auto_mappr_config.dart:58]: Fixed.

File: packages/auto_mappr/lib/src/builder/map_bodies/class_body_builder.dart (hunk 2 above)
Comment (tenhobi) [reply packages/auto_mappr/lib/src/builder/map_bodies/class_body_builder.dart:160]: Fixed, we can use the first fieldMapping

File: packages/auto_mappr/lib/src/models/auto_mappr_config.dart (hunk 4 above)
Comment (tenhobi) [reply packages/auto_mappr/lib/src/models/auto_mappr_config.dart:56]: Fixed.

**Review by petrnymsa — APPROVED** [review]

**Pull request merged**